### ISSUE TITLE
Round Trip?

### ISSUE THREAD

**Issue by tbrooke** [kiranshila/cybermonday#10]
Can I do a round trip from my AST representation back to the original Markdown instead of HTML?

**Comment by kiranshila** [comment]
I suppose it is possible - but not currently implemented. Is your use case to programmatically modify a markdown document via its AST?

**Comment by tbrooke** [comment]
Sorry for not following up - was thinking of modifying via AST but just doing modifications as markdown -- So you are right really don't need it

**Comment by tbrooke** [comment]
I'm back after deciding I do need ir->md to change AST for a new different markdown doc -- I forked it and may try, will do a pull request if succeed - I studied your code some,  it seems doable and maybe not that difficult - any thoughts or suggestions are welcome --

**Comment by kiranshila** [comment]
I agree this is useful. I would work on this, but am in the process of moving. It shouldn't be too bad, we can copy the lowering to hiccup part to just emit strings.
